actions/ci-test-ts/action.yml

```diff
@@ -71,6 +71,15 @@ runs:
       shell: bash
       run: pnpm run test:affected
 
+    - name: Store test result artifacts
```

Comment:
I don't see any of the repos using this action creating any format of report, so I'm assuming junit as the standard and going with it